### PR TITLE
Release/uofu master cas

### PR DESCRIPTION
**Pull request by loganfreeman**
I updated the branch to v3.0.0 and recreated the pull request.

### CODE REVIEW

File: config/initializers/01_obis_setup.rb [hunk 1]
```diff
@@ -90,6 +90,11 @@
   USE_FEEDBACK_LINK                         = application_config['use_feedback_link'] || false
   FEEDBACK_LINK                             = application_config['feedback_link'] || "#"
   NAVBAR_LINKS                              = application_config['navbar_links'] || {}
+  USE_CAS                                   = application_config['use_cas'] || false
```
Comment (kyle-glick) [inline config/initializers/01_obis_setup.rb]: @loganfreeman Can you add the corresponding constants to the application.yml.example file:
`use_cas`
`use_cas_only`
Also note: We will need to add these to the OS Settings branch dataset

Comment (loganfreeman) [reply config/initializers/01_obis_setup.rb]: Kyle:

I added use_cas and use_cas_only to application.yml.example

Comment (kyle-glick) [reply config/initializers/01_obis_setup.rb]: Thanks!

**Comment by Stuart-Johnson** [comment]
https://www.pivotaltracker.com/story/show/151338155

**Pull request merged**